components/provisioner/e2e_test/test.sh

```diff
@@ -33,6 +33,12 @@ function has_to_succeed {
 function wait_for {
     echo "Waiting $1 to launch on $2"
 
+
+    if ! which timeout ; then
+        printf '\n Please install timeout; e.g. in MacOs you can use `brew install coreutils` \n\n'
+        exit 2
```

Comment:
All the other `exit`s use code `1`, why use `2` for this one?
Either each should be different or all should be the same imo

Reply:
/hold
so you can read the comment before merging

Reply:
I've changed that exist code to `1`